qutebrowser/config/config.py

```diff
@@ -731,28 +732,49 @@ def set_command(self, win_id, section_=None, option=None, value=None,
             print_ = True
         else:
             with self._handle_config_error():
-                if option.endswith('!') and option != '!' and value is None:
+                if option.endswith('!') and option != '!' and len(values) == 0:
```

Comment:
Yes - see "*For sequences, (strings, lists, tuples), use the fact that empty sequences are false.*" in [PEP8](https://www.python.org/dev/peps/pep-0008/) (the commonly agreed-upon Python styleguide).

What I think is an issue is checking falsiness when checking if something is `None`. I'd say a name usually has a type which doesn't change (e.g. `values` will always be a sequence), but it's somewhat common to have a name to be either `None` or a "real" value (i.e. `Optional[Sequence[str]]` in [PEP484 annotation](https://www.python.org/dev/peps/pep-0008/)).

In the first case (like here), we're certain it's a sequence, and we want to check whether it's empty or not, so using its truthyness seems fine to me. In the second case, it's more confusing when reading (I'd assume you'd check for an empty sequence), and it's easy to accidentally also check for an empty sequence when you actually only wanted to check for `None`.

(A sequence is e.g. a list/tuple - but they all quack like a duck :wink:)